README.md

```diff
@@ -301,6 +301,45 @@ import exampleGo from '!!raw-loader!./code-example.go'
 ```
 ````
 
+### Mermaid graphs
+
+Use [mermaid.js](https://mermaid-js.github.io/mermaid/#/) to create graphs and
+diagrams,
+[for example](https://www.ory.sh/docs/kratos/self-service/flows/user-registration).
+The [mermaid live editor](https://mermaid.live) runs in the browser.  
```

Comment:
Ok, cool. How about a sentence that explains why you reference this tool? Sth like "you can create diagrams manually or use a convenient web-based tool like [tool]"? Instead of just dropping a link. Let's be pro.